tasktiger/worker.py

```diff
@@ -344,11 +429,9 @@ def _execute(self, queue, tasks, log, locks, all_task_ids):
         else:
             # Main process
             log = log.bind(child_pid=child_pid)
-            log.debug('processing', func=task_func, params=[{
-                    'task_id': task.id,
-                    'args': task.args,
-                    'kwargs': task.kwargs,
-            } for task in tasks])
+            for task in tasks:
+                log.info('processing', func=task_func, task_id=task.id,
+                         params={'args': task.args, 'kwargs': task.kwargs})
```

Comment:
Don't care too much about the log format, but why the change from one log message to multiple?